samples/global.param.sample.image.txt

```diff
@@ -163,20 +160,16 @@ SNOW_BAND	1	# Number of snow bands; if number of snow bands > 1, you must insert
 RESULT_DIR      (put the result directory path here)	# Results directory path
 OUTPUT_STEPS_PER_DAY        0       # Output interval (hours); if 0, OUTPUT_STEPS_PER_DAY = MODEL_STEPS_PER_DAY
 SKIPYEAR 	0	# Number of years of output to omit from the output files
-COMPRESS	FALSE	# TRUE = compress input and output files when done
-BINARY_OUTPUT	FALSE	# TRUE = binary output files
 ALMA_OUTPUT	FALSE	# TRUE = ALMA-format output files; FALSE = standard VIC units
 MOISTFRACT 	FALSE	# TRUE = output soil moisture as volumetric fraction; FALSE = standard VIC units
-PRT_HEADER	FALSE   # TRUE = insert a header at the beginning of each output file; FALSE = no header
 PRT_SNOW_BAND   FALSE   # TRUE = write a "snowband" output file, containing band-specific values of snow variables; NOTE: this is ignored if N_OUTFILES is specified below.
 
 #######################################################################
 #
 # Output File Contents
 #
-# As of VIC 4.0.6 and 4.1.0, you can specify your output file names and
-# contents # in the global param file (see the README.txt file for more
-# information).
+# You can specify your output file names and contents # in the global param
```

Comment:
FYI the # indicates that the remainder of the line is commented out, like "//" in C.